### ISSUE TITLE
Add users to Clients

### ISSUE THREAD

**Issue by AlexYoung28** [evolutionscript/HelpDeskZ-1.0#1]
Can you add clients and users to those clients?

**Comment by evolutionscript** [comment]
I didn't understand what you mean, or it's about if you can assign clients to staff users?

**Comment by AlexYoung28** [comment]
Sorry should of said more!

It would be nice to have say Client A, Client B etc.. and within those you can select User A who works for Client A, that way we can get reports on which clients logging the most tickets and also users as well?

Does that make sense?

**Comment by evolutionscript** [comment]
Ok, it is like assign clients to user staff, it is not implemented but it is a good idea for a next version

**Comment by mikechristopher** [comment]
I believe Alex is more looking for having companies/organisations and then having multiple customers from the one organisation being able to log tickets.  Is my understanding correct? @AlexYoung28

**Comment by AlexYoung28** [comment]
Yeah just i'm in a 3rd party IT company so we have lots of companies and users within those companies so being able to log tickets like that

**Comment by mikechristopher** [comment]
I agree with this also - I work for a software company of which have many companies as their customer base and then multiple people from those companies creating tickets on behalf of their company so believe this is a useful addition.  It would be good for any of the users within the company can view all of the companies tickets.

**Comment by Gliba** [comment]
+1

**Comment by donnyv** [comment]
+1

**Comment by razareal** [comment]
When we expect companies/organisations feature ?

**Comment by cassiohuber** [comment]
I'm waiting for this update too
